tf-deploy/myapp/main.tf

```diff
@@ -0,0 +1,27 @@
+# Example ECS service terraform config file.
+# See README.md for details on how to fill in all the commented out values.
+
+module "service" {
+  source           = "../../templates/basic-app"
+  desired_count    = "4"
+  vpc_name         = "ecs-example-vpc"
+  region           = "us-east-1"
+  #target_group_arn = "arn:aws:elasticloadbalancing:us-east-1:CHANGEME"
+  cluster          = "ecs-example-vpc-cluster-a"
+  docker_image     = "${var.docker_image}"
+  app_name         = "basic-app"
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+terraform {
```

Comment:
Do you want to mention state locking here? Seems like deployments may have a greater change of colliding than infra changes.